Clojure/ClojureScript.sublime-syntax

```diff
@@ -0,0 +1,14 @@
+%YAML 1.2
+---
+# https://www.sublimetext.com/docs/3/syntax.html
+
+name: ClojureScript
+scope: source.clojurescript
```

Comment:
Using `source.clojure.script` would be more desirable, as this is not only subfamily of clojure but actually the same syntax (at least currently). Linters explicitly targetting the version on Java could exclude `source.clojure.script` from their selector, as could Zeal, while features that apply to all languages of the clojure family (such as the comment preferences below) only need to target `source.clojure`.

Reply:
Thoughts on `source.clojure.clojurescript` or `source.clojure.cljs` vs `source.clojure.script`?

Reply:
I could see `source.clojure.clojurescript` as a more descriptive version, though I find it hard to believe that `source.clojure.script` would not be understood by anyone reading it. Still, I never used either. Definitely not `source.clojure.cljs`, though.

Reply:
`source.clojure.clojurescript` made sense to me. And it avoids having to modify plugins.